Team6220_2021/src/main/java/org/firstinspires/ftc/team6220_2021/MasterAutonomous.java

```diff
@@ -5,18 +5,34 @@
 import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
 import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
 import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
-import org.firstinspires.ftc.team6220_2021.ResourceClasses.PIDFilter;
+import org.firstinspires.ftc.team6220_2021.ResourceClasses.Constants;
 
+import java.util.Arrays;
 
 public abstract class MasterAutonomous extends MasterOpMode {
+    public double max(double a, double b, double c) {
+        double[] array = {a, b, c};
+        Arrays.sort(array);
```

Comment:
Sorting an array is a very expensive way to get the max. Try to find something simpler.